lib/ripgrep.ts

```diff
@@ -0,0 +1,40 @@
+/**
+ * An easy wrapper around ripgrep. Mostly to make install easier.
+ * @author mia-pi-git
+ */
+
+import * as Streams from './streams';
+import * as child_process from 'child_process';
+import {FS} from './fs';
+
+const ROOT = FS('./').path;
```

Comment:
😬

I don't know if this is supposed to stay stable like this...

Reply:
You only use it as a default `options.cwd`; I think it's fine just to grab it from FS every time.

Reply:
Alright.